pages/linux/kernel-install.md

```diff
@@ -0,0 +1,16 @@
+# kernel-install
+
+> Add and remove kernel and initrd images to and from /boot.
+> More information: <https://man.archlinux.org/man/kernel-install.8>.
+
+- Add kernel and initramfs images to bootloader partition:
+
+`sudo kernel-install add {{kernel-version}} {{kernel-image}} {initrd-file ...}`
```

Comment:
```suggestion
`sudo kernel-install add {{kernel-version}} {{kernel-image}} {{initrd-file ...}}`
```

Reply:
Seem like we both reviewed at same time 😅 , will resolve this suggestion alone. The other one LGTM.